GUIDE.md

```diff
@@ -0,0 +1,105 @@
+Dockers can be resized and their titles dragged around so you can find the best setup for you!
+
+All three dockers on the left serve the same purpose: showing you the classes in the project, just like you'd see in your IDE.
+Each class has a little icon to show how well-mapped it is: a green check for done, a yellow dot for in progress, and a red x for untouched.
+Double-click on packages and classes to open them. You can also right-click to see a context menu with some shortcuts to help you map faster!
+When you open a class, it will be decompiled and the source code shown in the main view in the centre.
+You'll immediately notice a couple things:
+1. Some names, or *tokens*, are highlighted in red, green, or grey. Red means a name isn't mapped, green means it *is* mapped, and grey means it's been
+   automatically mapped by our [enigma plugin](https://github.com/QuiltMC/quilt-enigma-plugin). Don't be afraid of editing green or grey names,
+   there's always something better to find!
+2. The file is in a tab: Enigma supports having multiple files decompiled and ready to access at once. Nice!
+3. The "structure" docker on the right has updated! We'll explain how to use that soon.
+
+## Naming
+
+The process of naming boils down to four steps:
+1. Read the relevant code. The first step in coming up with a name should always be reading and brainstorming your own names.
+   For every type of token, you can use the "show calls" tool on right-click to find usages if you're lacking in code to assess.
+2. Reread some code! Taking a look at the code with your name applied will help you get a good idea of the quality of your name.
+3. Cross-reference your name. QM mappers have access to Yarn and Mojmap to compare their names against, using tools like [Linkie](https://linkie.shedaniel.me/mappings)
+   and [WagYourTail's mapping viewer](https://wagyourtail.xyz/Projects/MinecraftMappingViewer/App). If any other name contradicts yours, triple-check your name to make
+   sure you haven't made a mistake! After checking the other names, hybridize, copy, or keep your original name to find the best possible option.
+4. Consider Javadoc. If your name is unclear and you can't improve it without making it too wordy, throw in some Javadoc to make sure modders know
+   what they're looking at.
+
+### Tips: naming fields
+
+With fields, "show calls" is your best friend. You'll want to run through many of these before finalising your decision.
+Constants will often appear to have no usages, which is misleading: the compilation has inlined its value instead of using a reference.
+Use `ctrl + f` to find what you're looking for, and reference the Mojang name since Yarn will often fail to map these.
+
+### Tips: naming classes
+
+Class names should come with plenty of deliberation, given that they're the most visible of all.
+Make sure to use the "show inheritance" and "show implementations" tools on right-click to make sure the name you're choosing isn't too granular.
```

Comment:
```suggestion
Make sure to use the "show inheritance" and "show implementations" tools on right-click to make sure the name you're choosing is appropriately broad/narrow.
```
Not 100% on this, but seems like a more complete description of what to look out for imo.